Workflow/Sources/RenderContext.swift

```diff
@@ -71,10 +71,6 @@ public class RenderContext<WorkflowType: Workflow>: RenderContextType {
         fatalError()
     }
 
-    public func awaitResult<W, Action>(for worker: W, outputMap: @escaping (W.Output) -> Action) where W: Worker, Action: WorkflowAction, WorkflowType == Action.WorkflowType {
```

Comment:
Are we not providing a backwards-compatible deprecated shim for these?

Reply:
I was thinking not, but I'll go ahead and add it in. We'll still need to do the `import WorkflowReactiveSwift`, but I'll expose the same APIs there.

Reply:
Done.